src/Symfony/Installer/NewCommand.php

```diff
@@ -36,6 +36,12 @@
 class NewCommand extends Command
 {
     /**
+     * Constants.
+     */
+    const VERSION_LATEST = 'latest';
+    const VERSION_LTS = 'lts';
```

Comment:
The idea was to be able to deal with symfony download

https://github.com/symfony/symfony-installer/blob/master/src/Symfony/Installer/NewCommand.php#L218

http://symfony.com/download?v=Symfony_Standard_Vendors_latest => _can_ be resolved
http://symfony.com/download?v=Symfony_Standard_Vendors_lts => _can not_ be resolved (exception here https://github.com/symfony/symfony-installer/blob/master/src/Symfony/Installer/NewCommand.php#L270)